src/components/views/rooms/MemberList.js

```diff
@@ -136,11 +136,13 @@ export default class MemberList extends React.Component {
     _getMembersState(members) {
         // set the state after determining _showPresence to make sure it's
         // taken into account while rerendering
+        const cli = MatrixClientPeg.get();
         return {
             loading: false,
             members: members,
             filteredJoinedMembers: this._filterMembers(members, 'join'),
             filteredInvitedMembers: this._filterMembers(members, 'invite'),
+            canInvite: cli.getRoom(this.props.roomId).canInvite(cli.getUserId()),
```

Comment:
would be good to null-guard the room, other places in the code seem to as well

Reply:
Okay I'll modify